components/ws-daemon/pkg/controller/snapshot_controller.go

```diff
@@ -124,5 +128,20 @@ func (ssc *SnapshotReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 		log.Error(err, "could not set completion status for snapshot", "workspace", snapshot.Spec.WorkspaceID)
```

Comment:
unrelated to the PR, but would be nice to log the snapshotErr here if it fails to update the status